10_datastructures_lab/01.py

```diff
@@ -1,10 +1,15 @@
-"""
-Write a program that takes two strings
-from the user and checks if they represent
-a valid user name.
-Valid users and passwords:
-    apple => red
-    lettuce => green
-    lemon => yellow
-    orange => orange
-"""
+import sys
+hosts = {}
+
+#read all hosts from file
+with open('4.Collections\\hosts', 'r') as f:
+    for line in f:
+        keyvalue = line.split("=")

```

Comment:
How about: `name, ip = line.rstrip().split('=')`
Now you don't need the indexes in the next line and everything's easier to read